package/gluon-mesh-vpn-wireguard/files/etc/init.d/gluon-mesh-vpn-wireguard

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh /etc/rc.common
+START=45
+USE_PROCD=1
+
+secretfile=/lib/gluon/mesh-vpn/wireguard
```

Comment:
The secret key should reside in UCI.

Reply:
conflicting comment: https://github.com/freifunk-gluon/gluon/pull/1534#discussion_r352795808

Reply:
this pattern was discussed with neoraider on irc a year back. Since wireguard needs the secret in a file and since this is used in multiple places, it was deemed sensible to write it to a file instead.

I agree though that the file should be read from uci instead of hard-coded.